code/android-core-library/src/phone/java/com/adobe/marketing/mobile/MobileCore.java

```diff
@@ -200,6 +203,39 @@ public static void log(@NonNull final LoggingMode mode, final String tag, final
         }
     }
 
+    public static void registerExtensions(@NonNull final List<Class<? extends Extension>> extensions, final AdobeCallback<?> completionCallback) {
+        if(!sdkInitializedWithContext.get()) {
+            Log.error(LOG_TAG, "Failed to registerExtension - setApplication not called");
+            return;
+        }
+
+        List<Class<? extends Extension>> allExtensions = new ArrayList<>();
+        // allExtensions.add(ConfigurationExtension.class);
+        if (extensions != null) {
+            for(final Class<? extends Extension> extension: extensions) {
+                if (extension != null) {
+                    allExtensions.add(extension);
+                }
+            }
+        }
+
+        AtomicInteger registeredExtensions = new AtomicInteger(0);
+        for (final Class<? extends Extension> extension: allExtensions) {
+            EventHub.Companion.getShared().registerExtension(extension, eventHubError -> {
+                Log.debug(LOG_TAG, "Finished registering extension " + extension + "with status "+ eventHubError);
```

Comment:
nit: get rid of "Finished" here, it's confusing with the log statement below.  "Registered extension" would be better.